bluerov2_control/ControlMode mode
---
bool success
